web/docs/migrate-from-0-11-to-0-12.md

```diff
@@ -373,7 +561,7 @@ You should see the new `Auth`, `AuthIdentity` and `Session` tables in your datab
 1. **Update your JS/TS code** to work correctly with the new auth.
 
   You should use the new auth helper functions to get the `email` or `username` from a user object. For example, `user.username` won't work anymore, now you need to do `getUsername(user)`. Read more about the helpers in the [Auth Entities - Accessing the Auth Fields](auth/entities#accessing-the-auth-fields) section. The helpers you are most likely to use are the `getEmail` and `getUsername` helpers.
-1. Finally, **check that your app now fully works as it worked before**. If all the above steps were done correctly, everything should be working now 🎉!
+1. Finally, **check that your app now fully works as it worked before**. If all the above steps were done correctly, everything should be working now.
```

Comment:
```suggestion
1. Finally, **check that your app now fully works as it worked before**. If all the above steps were done correctly, everything should be working now 🎉!
```

Reply:
In our docs writing guide we discourage the use of emojis: https://github.com/wasp-lang/wasp/blob/main/web/docs/writingguide.md#grammar

Reply:
I know we do. That said, we are celebrating victory here, it is a big moment, they migrated the script, and emoji makes that clear, plus is a visual anchor for a special moment.

We are trying to not use emojis to not clutter the text and to keep things professional, but banishing every single usage of emoji ever is taking the rule to the absurd.

Reply:
I believe what we wrote in the guide makes sense and it's not taking it to the absurd. 

> Don't use emojis (except in discussions). Emojis are cute and friendly, but they can be a distraction in documentation. Some emojis even convey different meanings in different cultures. They also make the documentation seem unprofessional and of lower quality.

Reply:
Martin, rules :book: are the only thing separating us :people_holding_hands: from animals :wolf: